docs/pages/eas-update/migrate-from-classic-updates.mdx

```diff
@@ -8,6 +8,8 @@ import { Step } from '~/ui/components/Step';
 
 EAS Update is the next generation of Expo's updates service. If you're using Classic Updates, this guide will help you upgrade to EAS Update.
 
+> SDK 49 will be the last version to support Classic Updates. To continue using the deprecated `expo publish` command, set [`updates.useClassicUpdates`](versions/latest/config/app/#useclassicupdates) in your app config.
```

Comment:
```suggestion
> SDK 49 will be the last version to support Classic Updates. To continue using the deprecated `expo publish` command, set [`updates.useClassicUpdates`](/versions/latest/config/app/#useclassicupdates) in your app config.
```

- Missing `/`
- Just a suggestion (should not block this PR) but I think we should move this note before line no. 9 (that is, at the top of this doc).